pkg/network/image-pull-policy.go

```diff
@@ -0,0 +1,54 @@
+package network
```

Comment:
can you add license here?

Reply:
we don't have the license header in any of the other modules. we should add it in another PR